### ISSUE TITLE
Handling missing total budget

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-backend#53]
On https://kohesio.eu/projects/Q154346 for instance we see

Total budget: 0 €
Funded by EU: 4 000 000 € (0%)

Should be:

Total budget: **Unknown**
Funded by EU: 4 000 000 €

Should we handle this in the API and/or UI?

**Comment by D063520** [comment]
The problem I guess is that is written in the original data ... so what to do? Do we remove all budgets that are 0? Like here:
https://linkedopendata.eu/wiki/Item:Q154346
There are currently more then 100 projects like this:
https://query.linkedopendata.eu/#%20SELECT%20%3Fs%20%3Fo%20%20WHERE%20%7B%09%0A%20%3Fs%20%3Chttps%3A%2F%2Flinkedopendata.eu%2Fprop%2Fdirect%2FP474%3E%20%20%3Fo%20.%0A%20%20%20FILTER%20%28%3Fo%20%3D%200%29%0A%20%0A%20%7D

**Comment by madewild** [comment]
Yes good point but it's obviously wrong... so yes I would rather remove these statements from the Wikibase altogether and add a filter to prevent zero amounts when importing new projects.

**Comment by D063520** [comment]
Ok, I assign this to Ali. He is not here today, but he can do it tomorrow.

**Comment by AlyHdr** [comment]
I removed all the statements with 0 budget from the wikibase, but now in the frontend instead of having unknown it shows a blank. So @raphdom do you prefer that we return "unknown" from the API or you could put unknown whenever the value is "" ?

**Comment by raphdom** [comment]
I can do it on my side.
